### ISSUE TITLE
Rename bud to build

### ISSUE THREAD

**Issue by Fodoj** [containers/buildah#3366]
This is more of a suggestion that the issue, of course. 

`bud` is `build-using-dockerfile`. This command will also understand 'Containerfile', which is not Docker-oriented, at least from the name. Renaming `bud` to just `build` will remove this kind of discrepancy. And once Containerfile is a real thing, and not just a convention, then `build` will make ever more sense, than `bud`.

**Comment by rhatdan** [comment]
Interested in opening a PR to make bud == build, so either would work.  I don't see us removing bud, since it is well known and would break backwards compatibility.

**Comment by Fodoj** [comment]
Yes, I agree, for backwards compatibility bud needs to stay for a while.
